fastlane/lib/fastlane/actions/apple_pay_cert.rb

```diff
@@ -0,0 +1,153 @@
+        end
+
+        if params[:generate_p12]
+          p12_cert_path = File.join(output_path, "#{filename_base}.p12")
+          p12 = OpenSSL::PKCS12.create(params[:p12_password], 'production', pkey, x509_certificate)
+          File.write(p12_cert_path, p12.to_der)
+          UI.message("p12 certificate: ".green + Pathname.new(p12_cert_path).realpath.to_s)
+        end
+
+        x509_cert_path = File.join(output_path, "#{filename_base}.pem")
+        File.write(x509_cert_path, x509_certificate.to_pem + pkey.to_pem)
+        UI.message("PEM: ".green + Pathname.new(x509_cert_path).realpath.to_s)
+        return x509_cert_path
+      end
+
+      def self.certificate
+        Spaceship.certificate.apple_pay_certificate
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.is_supported?(platform)
+        [:ios].include?(platform)
+      end
+
+      def self.description
+        'Creates Apple Pay certificate'
+      end
+
+      def self.details
+        "This action allows you to create Apple Pay merchant certificate."
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :username,
+                                       env_name: "APPLE_PAY_USERNAME",
+                                       description: "Your Apple ID Username"),
+          FastlaneCore::ConfigItem.new(key: :merchant_bundle_id,
+                                   env_name: "APPLE_PAY_MERCHANT_BUNDLE_ID",
+                                   description: "You merchant bundle identifier (e.g: merchant.com.example)"),
+          FastlaneCore::ConfigItem.new(key: :team_id,
+                                 short_option: "-b",
+                                 env_name: "APPLE_PAY_CERT_TEAM_ID",
+                                 code_gen_sensitive: true,
+                                 default_value: CredentialsManager::AppfileConfig.try_fetch_value(:team_id),
+                                 default_value_dynamic: true,
+                                 description: "The ID of your Developer Portal team if you're in multiple teams",
+                                 optional: true,
+                                 verify_block: proc do |value|
+                                   ENV["FASTLANE_TEAM_ID"] = value.to_s
+                                 end),
+          FastlaneCore::ConfigItem.new(key: :p12_password,
+                                     short_option: "-p",
+                                     env_name: "APPLE_PAY_CERT_P12_PASSWORD",
+                                     sensitive: true,
+                                     description: "The password that is used for your p12 file",
+                                     default_value: ""),
+          FastlaneCore::ConfigItem.new(key: :team_name,
```

Comment:
shouldn't team_id and team_name be next to each other?

Reply:
still unanswered - as this defines the order the params are displayed in the docs, this is important

Reply:
No problem. Moved it.